driver-core/build.gradle

```diff
@@ -58,19 +58,17 @@ afterEvaluate {
     jar.manifest.attributes['Import-Package'] = [
         'org.bson.*', // unfortunate that this is necessary, but if it's left out then it's not included
         'javax.crypto.*',
-        'javax.crypto.spec.*',
```

Comment:
These are not needed as java.crypto.* already handles the inclusion.

Reply:
Glad you were able to confirm this. That warning bothered me.